### ISSUE TITLE
bnd's non-workspace gradle plugin does not support bnd merge properties

### ISSUE THREAD

**Issue by rotty3000** [bndtools/bnd#1171]
bnd's non-workspace gradle plugin uses the standard gradle jar task's manifest configuration for setting bnd properties.

However, that implementation validates header key syntax which does not support bnd's notion of merge properties, which contain periods (e.g. `-property.*`).

e.g. having something like:
```
jar {
	manifest {
		attributes "-thekey.thesuffix": "thevalue"
	}
}
```

results in an error like:
```
* What went wrong:
A problem occurred evaluating script.
> The Key=-thekey.thesuffix violates the Manifest spec!

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED
```

**Comment by bjhargrave** [comment]
I am not sure what you want here. You are complaining about the standard Gradle manifest object. bnd is not going to change that.

The [gradle validation](https://github.com/gradle/gradle/blob/master/subprojects/platform-jvm/src/main/java/org/gradle/api/java/archives/internal/DefaultAttributes.java#L57) uses <https://docs.oracle.com/javase/7/docs/api/java/util/jar/Attributes.Name.html#Attributes.Name%28java.lang.String%29> which does not even appear to be fully correct. It complains about `-thekey.thesuffix` because of the `.` but fails to complain about the leading `-` which is also invalid per the [manifest spec](https://docs.oracle.com/javase/7/docs/technotes/guides/jar/jar.html#Name-Value_pairs_and_Sections).

Basically, you must then use a bnd file to hold [bnd instructions](http://bnd.bndtools.org/chapters/820-instructions.html) which gradle wont allow in the manifest. The standard jar task will use the `bnd.bnd` file without any further  configuration once the bnd plugin is applied.

**Comment by rotty3000** [comment]
Yeah, furthermore, even to set global "defaults" I am fine with using an `-include` which at least is syntactically acceptable to the manifest validation.

e.g. `attributes "-include": bndGlobalDefaultsFilePath()
